modules/@ergonode/core/src/layouts/default.vue

```diff
@@ -92,34 +92,30 @@ export default {
             this.breadcrumbs = this.$route.meta.breadcrumbs || [];
         },
         batchActions() {
-            const request = [];
+            const requests = [];
 
             this.batchActions.forEach(({
                 id,
-                href,
-                type,
-                payload,
+                request,
             }) => {
                 if (!this.executingBatchActions[id]) {
                     let event = null;
 
                     this.executingBatchActions[id] = true;
 
-                    request.push(
-                        this.$axios.$post(href, {
-                            type,
-                            ...payload,
-                        }).then(() => {
+                    requests.push(
+                        this.$axios[request.type](request.href, request.payload).then(() => {
```

Comment:
Can we move this request to `services` ? To keep all `axios` requests in one place.